app/scripts/controllers/app-state-controller.ts

```diff
@@ -147,18 +156,16 @@ type AppStateControllerInitState = Partial<
   >
 >;
 
-type AppStateControllerOptions = {
+export type AppStateControllerOptions = {
   addUnlockListener: (callback: () => void) => void;
   isUnlocked: () => boolean;
-  initState?: AppStateControllerInitState;
+  state?: AppStateControllerInitState;
```

Comment:
I see that this was typed incorrectly already, but presumably what we want here is a partial state (as is conventional for initial state in all controllers)

```suggestion
  state?: Partial<AppStateControllerInitState>;
```

Reply:
Sorry about that, I misread the type. We can undo this change. This is already a `Partial`, see that partial is part of the `AppStateControllerInitState` type. I thought this was `AppStateControllerState`.